core/site.go

```diff
@@ -77,10 +79,12 @@ type Site struct {
 	savings     *Savings                 // Savings
 
 	// cached state
-	gridPower    float64 // Grid power
-	pvPower      float64 // PV power
-	batteryPower float64 // Battery charge power
-	batterySoc   float64 // Battery soc
+	gridPower     float64 // Grid power
+	pvPower       float64 // PV power
+	pvEnergy      float64 // PV total energy
+	batteryPower  float64 // Battery charge power
+	batteryEnergy float64 // Battery total discharge energy
```

Comment:
Ist das hier wirklich nur die aus der Batterie entnommene Energie? In Zeile 470 wird ja unabhängig des Vorzeichens aufaddiert `site.batteryEnergy += energy`. Wenn man hier weitere Größen einführt, dann mag es für die Statistik in der Tat interessant sein separat die in die Batterie eingespeicherte Energie und die aus der Batterie entnommene Energie zu erfassen.
Wann werden die Zähler zurückgesetzt?

Reply:
Wir fragen die geladene Energie nirgends ab, ist in keinem Template berücksichtigt. Genauso wenig wie die ins Netz exportierte Energie.
Die aus der Batterie entnommene Energie ist hier nur ein Beifang weil "eh da".
Den Rest kann man notfalls errechnen.

Wer es noch detaillierter braucht muss selber ran.

Das hier sind nur noch die mit einfachen Mitteln machbaren Werte.